### PR TITLE
Build infra project setup and Gitlab Runner installation scripts

### PR DESCRIPTION
**Pull request by ivanmrva**

### CODE REVIEW

File: group_vars/all.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  api_version: v1
+  username: "{{ os_username }}"
+  auth_token: "{{ os_auth_token }}"
+
+project:
+  name: build-infra
+
+gitlab_runner:
+  image_name: gitlab/gitlab-runner
+  version: latest
```
Comment (fabiangotzen) [inline group_vars/all.yml]: Should we use latest for this?

File: README.md [hunk 2]
```diff
@@ -1 +1,79 @@
...
+
+## Deploy the build infrastructure
+
+1. Before executing the deployment scripts, the following dependencies have to be installed on the machine,
+where the Ansible playbook will be executed:
+    * `pip install openshift`
+1. Start the deployment of the complete Openshift build infrastructure project as follows:
+    * `ansible-playbook -i <inventory_name> install-build-infra.yml --vault-id=<path-to-pwd-file>`
+    * where _<inventory_name>_ is the target environment (i.e. "dev", "prod", etc.),
+    * where _vauld-id_ parameter requires a path to file containing the Vault password (not pushed in the repository),
```
Comment (fabiangotzen) [inline README.md]: typo vauld-id

Comment (ivanmrva) [reply README.md]: fixed

File: group_vars/all.yml (hunk 1 above)
Comment (ivanmrva) [reply group_vars/all.yml]: added a comment and extended readme

**Pull request merged**